package.json

```diff
@@ -13,11 +13,12 @@
   "author": "David Furnes <dfurnes@dosomething.org>",
   "license": "MIT",
   "dependencies": {
-    "babel-preset-env": "^1.6.1",
+    "@babel/plugin-proposal-class-properties": "^7.0.0-beta.40",
```

Comment:
Is it safe to install `7.x`  plugins with Babel 6? Might want to drop to `^6.0.0` to be safe.

Reply:
Yeah, I was wondering about that too 😬 Okie dokie.